src/main/java/org/zalando/nakadi/domain/SubscriptionBase.java

```diff
@@ -29,6 +29,7 @@
     private Set<String> eventTypes;
 
     @NotNull
+    @Size(min = 1, message = "must contain at least one character")
```

Comment:
@v-stepanov because the default max value is 2147483647, in my opinion it is too much for consumer group. And it is good practice to validate borders of the input.